### PR TITLE
Fix Warnings in ScopedStorageService.kt

### PR DESCRIPTION
**Pull request by jDaowy**
Fixed warnings in ScopedStorageService.kt.
issue: #13282

## Purpose / Description
Fix AndroidStudio warnings #13282

## Fixes
Fixes part of #13282, specifically warnings in ScopedStorageService.kt

## Approach
Removes the warnings

## How Has This Been Tested?

Tested on physical device (Pixel 5)

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/ScopedStorageService.kt [hunk 1]
```diff
@@ -37,6 +35,8 @@ import com.ichi2.utils.FileUtil.isDescendantOf
 import com.ichi2.utils.Permissions
 import timber.log.Timber
 import java.io.File
+// import com.ichi2.anki.model.DiskFile
+// import com.ichi2.anki.model.RelativeFilePath
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/ScopedStorageService.kt:40]: just remove the line, don't uncomment

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/ScopedStorageService.kt [hunk 2]
```diff
@@ -48,6 +48,7 @@ typealias CollectionFilePath = String
 fun AnkiDroidDirectory.getCollectionAnki2Path(): CollectionFilePath =
     File(this.directory, CollectionHelper.COLLECTION_FILENAME).canonicalPath
 
+/* AnkiDroidDirectory.getRelativeFilePath() does not end up getting used
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/ScopedStorageService.kt:51]: Remove the code, we have `git` for history

**Review by david-allison — COMMENTED** [review]
Thanks! Few nitpicks, then good to go

**Review by lukstbit — APPROVED** [review]
Thanks for contributing!

**Review by david-allison — APPROVED** [review]
Perfect, awaiting CI. If there are further problems, rebase onto `main`, as I've just fixed a few  CI flakiness issues.

**Pull request merged**